contrib/gdk-pixbuf/loader.c

```diff
@@ -466,7 +466,7 @@ static gboolean avif_image_saver(FILE          *f,
     encoder->maxQuantizer = max_quantizer;
     encoder->minQuantizerAlpha = 0;
     encoder->maxQuantizerAlpha = alpha_quantizer;
-    encoder->speed = 7;
+    encoder->speed = 6;
```

Comment:
Yes, I recommend that we use the same speed here as the default encoder speed in avifenc.c (which is 6 right now).